advanced_functionality/mxnet_mnist_byom/mxnet_mnist.ipynb

```diff
@@ -223,11 +227,12 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
+    "import logging\n",
+    "logging.getLogger().setLevel(logging.WARNING)\n",
+    "\n",
     "predictor = sagemaker_model.deploy(initial_instance_count=1,\n",
     "                                          instance_type='ml.c4.xlarge')"
```

Comment:
Sorry I missed this point in the last review... Can you change this to an ml.m4.xlarge?  They are eligible for our free tier in hosting, so are the preferred instance type for example notebooks.